contracts/vaults/RibbonThetaVault.sol

```diff
@@ -45,6 +56,13 @@ contract RibbonThetaVault is DSMath, GnosisAuction, OptionsVaultStorage {
     // https://github.com/opynfinance/GammaProtocol/blob/master/contracts/MarginPool.sol
     address public immutable MARGIN_POOL;
 
+    // Our library for all Opyn related logic
+    address public immutable GAMMA_PROTOCOL;
```

Comment:
We don't actually need to store a reference to the GAMMA_PROTOCOL because it's already implicitly done by linking the library. We can just do `GammaProtocol.createShort` directly too I think

Reply:
yep, changed